rosdep/python.yaml

```diff
@@ -9990,6 +10009,10 @@ python3-sphinx-autoapi-pip:
   ubuntu:
     pip:
       packages: [sphinx-autoapi]
+python3-sphinx-copybutton-pip:
+  '*':
+    pip:
+      packages: [sphinx-copybutton]
```

Comment:
sphinx-copybutton is available on ubuntu and debian. Please create a `python3-sphinx-copybutton` key instead of a pip key.

https://launchpad.net/ubuntu/+source/sphinx-copybutton